client/rpc/debug/src/lib.rs

```diff
@@ -301,49 +303,53 @@ where
 						disable_stack,
 					} => {
 						if runtime_version.spec_version >= V2_RUNTIME_VERSION && api_version >= 3 {
```

Comment:
We should create a constant that holds at which runtime version we moved from old substrate (auto initialize block) to new one (not initializing block), so we can manually initialize or not. The rest should be the same across runtimes.

Reply:
Looks like it was this one https://github.com/PureStake/moonbeam/releases/tag/runtime-159

Reply:
Update dependencies to v0.9.8 series (#587)